scenario_configs/librispeech_entailment_asr.json

```diff
@@ -0,0 +1,86 @@
+{
+    "_description": "Baseline DeepSpeech ASR on LibriSpeech, contributed by MITRE Corporation",
+    "adhoc": {
+        "skip_adversarial": false
+    },
+    "attack": {
+        "knowledge": "white",
+        "kwargs": {
+            "batch_size": 1,
+            "eps": 10,
+            "eps_step": 0.5,
+            "max_iter": 10,
+            "norm": "snr",
+            "num_random_init": 0,
+            "targeted": true
+        },
+        "module": "armory.art_experimental.attacks.snr_pgd",
+        "name": "SNR_PGD_Numpy",
+        "targeted": true,
+        "targeted_labels": {
+            "kwargs": {
+                "dtype": "str",
+                "import_from": "armory.attacks.librispeech_target_labels",
+                "values": "entailment_100"
+            },
+            "module": "armory.utils.labels",
+            "name": "ManualTargeter"
+        },
+        "use_label": false
+    },
+    "dataset": {
+        "batch_size": 1,
+        "eval_split": "test_clean",
+        "framework": "numpy",
+        "module": "armory.data.datasets",
+        "name": "librispeech",
+        "train_split": "train_clean100"
+    },
+    "defense": null,
+    "metric": {
+        "means": false,
+        "perturbation": "snr_db",
+        "record_metric_per_sample": true,
+        "task": [
+            "entailment",
+            "word_error_rate"
+        ]
+    },
+    "model": {
+        "fit": false,
+        "fit_kwargs": {
+            "nb_epochs": 20000
+        },
+        "model_kwargs": {},
+        "module": "armory.baseline_models.pytorch.deep_speech",
+        "name": "get_art_model",
+        "predict_kwargs": {
+            "transcription_output": true
+        },
+        "weights_file": null,
+        "wrapper_kwargs": {
+            "clip_values": [
+                -1,
+                1
+            ],
+            "pretrained_model": "librispeech"
+        }
+    },
+    "scenario": {
+        "kwargs": {},
+        "module": "armory.scenarios.audio_asr",
+        "name": "AutomaticSpeechRecognition"
+    },
+    "sysconfig": {
+        "docker_image": "twosixarmory/pytorch-deepspeech",
+        "external_github_repo": [
+            "SeanNaren/deepspeech.pytorch@V3.0"
+        ],
+        "gpus": "all",
+        "local_repo_path": null,
+        "num_eval_batches": 100,
```

Comment:
is it intentional to have `"num_eval_batches"` in the config? This will error out when a `--check` flag is added

Reply:
Yes. The entailment labels are only defined for the first 100 samples.

Reply:
`--num-eval-batches` in the config will be overriden by CLI, so do we need to catch the case where the CLI dictates something > 100?

Reply:
I ran with `--num-eval-batches 105` and `--skip-attack` and got 

```
"benign_total_entailment": {
            "contradiction": 6,
            "entailment": 99,
            "total": 105
        },
```

Reply:
If the attack is active and `--num-eval-batches > 100`, the attack will error at runtime (after the 100th sample).

The benign results are not actually limited to 100 - those are with respect to the transcripts from the dataset.